release-1.8/release_notes_draft.md

```diff
@@ -72,6 +72,15 @@ fundamental aspect of a secure cluster.
   * The`--audit-policy-file` requires `kind` and `apiVersion` fields
     specifying what format version the `Policy` is using.
 
+* CronJobs has been promoted to `v1beta1` which is now turned on by default.
+  Although version `v2alpah1` is still available, it is deprecated and users are
+  encouraged to migrate to the new version.  Additionally, if you are running
+  cluster in high-availability configuration this might cause some troubles
+  when upgrading clusters since cronjob controller relies on the new version.
+
+* The `batch/v2alpha1.ScheduledJobs` has been removed, if you used it you need to
+  migrate to `batch/v1beta.CronJobs` instead.
```

Comment:
The `batch/v2alpha1.ScheduledJobs` has been removed.  Migrate to `batch/v1beta.CronJobs` to continue managing time based jobs.

Reply:
Updated.